vcbuild.bat

```diff
@@ -252,10 +254,24 @@ goto build-doc
 
 :project-gen
 @rem Skip project generation if requested.
-if defined noprojgen goto msbuild
+SETLOCAL EnableDelayedExpansion
```

Comment:
If this is only needed for L262 & L263, could this be moved there? Otherwise I can see this giving someone a headache in the future.

```batch
SETLOCAL EnableDelayedExpansion
set /p prev_configure_flags=<used_configure_flags
if NOT !prev_configure_flags! == !configure_flags! ENDLOCAL & goto run-configure
```